src/components/Listings/Listings.js

```diff
@@ -1,6 +1,25 @@
-import React, { Component } from 'react';
+import React from 'react';
 import './Listings.scss';
 
-export default Listings = () => {
-  return ()
-}
\ No newline at end of file
+const Listings = (props) => {
+  const filteredListings = props.listings.filter(listing => listing.area_id === props.selectedAreaId)
+  const domListings = filteredListings.map(listing => {
+    return (
+      <div className='listing-card'>
+        <h4>{listing.name}</h4>
+        <img src={`/images/${listing.listing_id}_a.jpg`} alt={listing.name} width='100px'/>
```

Comment:
I would avoid setting the `width` here. Add a `className` and adjust the styles in your stylesheet.

Reply:
Was a temporary setting because we were having issues getting sizing correct via other methods. I just fiddled with it a bit more and still having issues for some reason. Will push up what I was messing with now.